phantazm-api/src/main/java/com/github/phantazmnetwork/api/config/VariantConfigProcessor.java

```diff
@@ -42,6 +42,10 @@ public VariantConfigProcessor(@NotNull Function<Key, ConfigProcessor<? extends T
             throw new ConfigProcessException("no subprocessor");
```

Comment:
maybe include the key in the exception so it's easier to debug missing subprocessors